packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -333,6 +342,10 @@ export abstract class FluidDataStoreContext
 			this.mc.config.getBoolean(throwOnTombstoneUsageKey) === true &&
 			this._containerRuntime.gcTombstoneEnforcementAllowed &&
 			this.clientDetails.type !== summarizerClientType;
+
+		this.localChangesTelemetryThreshold = this.mc.config.getNumber(
+			"Fluid.Telemetry.LocalChangesTelemetryThreshold",
```

Comment:
I did not add a constant for this config name because it spans accross the container runtime and the data store runtime layers and I didn't think it was a good idea to add it to a common place (like runtime defintions).